src/main/flight/mixer_tricopter.h

```diff
@@ -73,4 +73,82 @@ int16_t triGetMotorCorrection(uint8_t motorIndex);
  */
 _Bool triEnableServoUnarmed(void);
 
+#ifdef USE_SERVOS
```

Comment:
These are all details of the tricopter mixer implementation right? Which now need to be visible in the unit tests.
To keep a clear view on what is part of our public API and what is implementation details I would like to propose to move internals to a separate header file, eg mixer_tricopter_internals.h.